DESCRIPTION.md

```diff
@@ -8,6 +8,10 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 
 # Release Notes
 
+- v3.12.0(TBD)
+  - Optimized `to_pandas()` performance by fully parallel downloading logic.
```

Comment:
is there any test result?
I'm wondering the perf gain in large vs small data set.
also any memory consumption change?

Reply:
I put some test result in the link above. I didn't change default thread number in this PR so there is no memory consumption change

Reply:
Here is the test result I recorded before: https://docs.google.com/spreadsheets/d/1JkFMBc8cTDpPrcUviNuKaKg2zdZqvIKguTo6PSJIJwo/edit?usp=sharing